bitrise.yml

```diff
@@ -0,0 +1,24 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  sample:
+    steps:
+    - path::./: { }
```

Comment:
NOTE: it would be great to have the sample workflow in a state from where we can easily test how the step works.